lib/oban/breaker.ex

```diff
@@ -45,6 +45,17 @@ defmodule Oban.Breaker do
     %{state | circuit: :enabled}
   end
 
+  @spec jitter(time :: pos_integer()) :: pos_integer()
+  def jitter(time) do
+    diff = trunc(:rand.uniform() * @jitter_mult * time)
```

Comment:
What if we pass the `mult` through as an option with a default? That will let us tweak the range from outside the module. The `jitter/2` function can then be rewritten using options; rather than a separate clause, it's an option that controls whether the jitter is applied to the leading, trailing, or both edges. WDYT?